### PR TITLE
Generate website from specs

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Generate website from specs with Hugo and with a Github Action at each push. The theme is the same used by Tests Scenarios (https://build.prestashop.com/test-scenarios/). The final url will be https://build.prestashop.com/prestashop-specs/
| Fixed ticket? | N/A

### CODE REVIEW

**Review by sowbiba — APPROVED** [review]
LGTM

File: config.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+baseURL: "http://localhost:1313/prestashop-specs/"
```
Comment (matks) [inline config.yml:1]: Hello, what is the scope of this PR? because this is not production-friendly 😄

File: config.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  geekdocLogo: "logo.png"
+  geekdocRepo: "https://github.com/PrestaShop/prestashop-specs"
+  geekdocSearch: true
+  geekdocSearchShowParent: true
+  geekdocMenuBundle: false
+
+markup:
+  goldmark:
+    renderer:
+      unsafe: true
```
Comment (matks) [inline config.yml:22]: Missing empty line

**Review by matks — COMMENTED** [review]
Hello, work seems overall ok :) bravo !

I asked a question about scope.

Also it would be great to provide a description a little more complete 😄  what hugo theme was chosen (license, version), why this one in particular, the setup with GitHub Actions...

Remember we're supposed to use [this Pull Request of @eternoendless](https://github.com/PrestaShop/PrestaShop/pull/16964) as an example of a good description 😊

File: config.yml (hunk 1 above)
Comment (Progi1984) [reply config.yml]: Fixed :p

File: config.yml (hunk 2 above)
Comment (Progi1984) [reply config.yml]: Fixed

**Comment by Progi1984** [comment]
@matks Done

**Comment by ttoine** [comment]
do you have already the future URL where it will be available?

**Comment by Progi1984** [comment]
@ttoine https://build.prestashop.com/prestashop-specs/

**Comment by eternoendless** [comment]
Dude! you could have continued where I left off in #218

Did you modify the original theme? Why not add it as a submodule?

**Comment by Progi1984** [comment]
@eternoendless The theme has been modified in the another repository so I take back the same. But as said by @PierreRambaud, we contact the UX team for a new theme after the release of this MVP.

**Review by PierreRambaud — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @PierreRambaud & @sowbiba

**Pull request merged**